### PR TITLE
Fix/3771  border active space should not move the title at sidebar

### PR DESCRIPTION
**Pull request by asf-algoo**
#3771 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: frontend/src/component/Sidebar/WorkspaceListItem.jsx [hunk 1]
```diff
@@ -94,8 +94,18 @@ class WorkspaceListItem extends React.Component {
...
+            // If the space is active the border width (5px) is subtracted
             props.level > 0 && (
-              <div style={{ marginLeft: `${(props.level - 1) * 20 + 10}px` }}>&#8735;</div>
+              <div
+                style={{
+                  marginLeft:
+                  `${(props.level - 1) * 20 + 10 -
+                    (props.location.pathname.includes(`${PAGE.WORKSPACE.ROOT}/${props.workspaceId}/`) ? 5 : 0)
+                  }px`
+                }}
```
Comment (raphj) [inline frontend/src/component/Sidebar/WorkspaceListItem.jsx:105]: Would it be possible to move this style in a class?

**Review by raphj — CHANGES_REQUESTED** [review]

File: frontend/src/component/Sidebar/WorkspaceListItem.jsx (hunk 1 above)
Comment (gbonaspetti) [reply frontend/src/component/Sidebar/WorkspaceListItem.jsx:105]: is there a way to pass the variables to the class? doesn't it make it more complicated knowing that we even have a conditional in it?

Comment (raphj) [reply frontend/src/component/Sidebar/WorkspaceListItem.jsx:105]: You are right. I'll try to think of an improvement for later.

Maybe put 10 and 20 in named constant though?

Comment (gbonaspetti) [reply frontend/src/component/Sidebar/WorkspaceListItem.jsx:105]: I don't think is needed here, normally I prefer constants, but in this case they are just number without a meaning, like:
const TEN = 10
const TWENTY = 20
`${(props.level - 1) * TWENTY + TEN - ...

**Review by raphj — APPROVED** [review]

**Pull request merged**